containers/datalab/Dockerfile.in

```diff
@@ -22,14 +22,17 @@ ADD content/ /datalab
 # Install the build artifacts
 RUN ln -s /datalab/web/static/datalab.css /datalab/nbconvert/datalab.css && \
     cd /datalab/web && /tools/node/bin/npm install && \
-    cd /
+    cd / && \
 
 # Install the support for connecting to a kernel gateway
-RUN wget https://github.com/jupyter/kernel_gateway_demos/archive/e653b2bd3ca91ef1af2a13c397766484dcb7b76d.zip && \
+    wget https://github.com/jupyter/kernel_gateway_demos/archive/e653b2bd3ca91ef1af2a13c397766484dcb7b76d.zip && \
     unzip -qq e653b2bd3ca91ef1af2a13c397766484dcb7b76d.zip -d kernel_gateway_demos && \
     rm e653b2bd3ca91ef1af2a13c397766484dcb7b76d.zip && \
     cd ./kernel_gateway_demos/kernel_gateway_demos-e653b2bd3ca91ef1af2a13c397766484dcb7b76d && \
-    pip install ./nb2kg/
+    pip install ./nb2kg/ && \
+
+# Clean up
+    apt-get purge -y wget unzip
```

Comment:
This shouldn't do anything, since it will only be removing files that were added in a lower image layer.